### PR TITLE
Change Core Philosophy

### PR DESCRIPTION
**Pull request by tlgimenes**
## What's the purpose of this pull request?
In the past months we have changed our philosophy. This PR moves the README closer to this philosophy

### CODE REVIEW

**Review by emersonlaurentino — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -44,14 +44,21 @@ The priorities are all about the stores' user experience and retailer satisfacti
...
+3. SEO - Stores should pass on all aspects of Google search console, like complete structured data and web vitals
 4. Feature completion - Enable you to craft your custom ecommerce experience
 
-## Why Jamstack and Gatsby?
+## Basic Architecture
 
-Jamstack focuses on performance and stability, which is part of our core philosophy.
+FastStore core is composed of two main packages:
+1. [`store-ui`](https://storeui.netlify.app/): A performant ecommerce ready component library
+2. `store-sdk`: An sdk for ecommerce to solve your basic ecommerce needs, like cart, pixel managment, session etc.
```
Comment (lariciamota) [inline README.md:54]:
```suggestion
2. `store-sdk`: An SDK for ecommerce to solve your basic ecommerce needs, like cart, pixel management, session etc.
```

File: README.md [hunk 2]
```diff
@@ -44,14 +44,21 @@ The priorities are all about the stores' user experience and retailer satisfacti
...
+
+These packages have bidings to popular frameworks, like Gatsby. These bindings are listed in this monorepo and help you create stores with Multiregion, Internationalization etc.
```
Comment (lariciamota) [inline README.md:56]:
```suggestion
These packages have bindings to popular frameworks, like Gatsby. These bindings are listed in this monorepo and help you create stores with Multiregion, Internationalization etc.
```

File: README.md [hunk 3]
```diff
@@ -27,7 +27,7 @@ FastStore is an open ecommerce framework built using modern technologies that ai
 - Jamstack based (deploy with Netlify, Vercel and any other JAMStack platform)
 - Marketplace ready with support to millions of skus
 - Intelligence via Google Analytics and GTM
-- Evergreen stores with our no-fork based approach
+- Evergreen stores with auto updates
```
Comment (victorhmp) [inline README.md:30]: How about "Evergreen stores that are always updated" or even "Evergreen stores that are automatically updated"?

File: README.md [hunk 4]
```diff
@@ -44,14 +44,21 @@ The priorities are all about the stores' user experience and retailer satisfacti
...
-## Why Jamstack and Gatsby?
+## Basic Architecture
 
-Jamstack focuses on performance and stability, which is part of our core philosophy.
+FastStore core is composed of two main packages:
+1. [`store-ui`](https://storeui.netlify.app/): A performant ecommerce ready component library
+2. `store-sdk`: An sdk for ecommerce to solve your basic ecommerce needs, like cart, pixel managment, session etc.
+
+These packages have bidings to popular frameworks, like Gatsby. These bindings are listed in this monorepo and help you create stores with Multiregion, Internationalization etc.
+We want to support other frameworks, like Next.JS and Create React App in the future
```
Comment (victorhmp) [inline README.md:57]: I'm not sure about listing Create React App as a framework. I would separate Next.JS from it. Something along the lines of: "We also want to support other React frameworks in the future, such as Next.JS, or even React applications bootstrapped using Create React App". 
Or not even mentioning Create React App at all 🤔  What do you think?

Comment (igorbrasileiro) [inline README.md:57]: @tlgimenes What do you think about creating a section with future features or something like that and put this phrase there?

**Review by igorbrasileiro — APPROVED** [review]
I don't have too much context about this new philosophy, but overall lgtm.

File: README.md [hunk 5]
```diff
@@ -44,14 +44,21 @@ The priorities are all about the stores' user experience and retailer satisfacti
 
 1. Performance - Great score on Lighthouse and Core Web Vitals
 2. Stability - E2E tests are complete and make sure the critical paths are covered
-3. No fork - Features should be delivered by dependencies and not by a store fork
+3. SEO - Stores should pass on all aspects of Google search console, like complete structured data and web vitals
 4. Feature completion - Enable you to craft your custom ecommerce experience
 
-## Why Jamstack and Gatsby?
+## Basic Architecture
```
Comment (Gmantiqueira) [inline README.md:50]: We don't have this section in table of contents/summary.

File: README.md [hunk 6]
```diff
@@ -7,7 +7,7 @@
   - [An open ecommerce framework](#an-open-ecommerce-framework)
   - [Why building FastStore?](#why-building-faststore)
   - [Project Philosophy](#project-philosophy)
-  - [Why Jamstack and Gatsby?](#why-jamstack-and-gatsby)
+  - [Why Jamstack?](#why-jamstack-and-gatsby)
```
Comment (tlgimenes) [inline README.md:10]:
```suggestion
  - [Why Jamstack?](#why-jamstack)
```

File: README.md (hunk 3 above)
Comment (tlgimenes) [inline README.md:30]:
```suggestion
- Evergreen stores
```

File: README.md [hunk 7]
```diff
@@ -44,14 +44,21 @@ The priorities are all about the stores' user experience and retailer satisfacti
...
-## Why Jamstack and Gatsby?
+## Basic Architecture
 
-Jamstack focuses on performance and stability, which is part of our core philosophy.
+FastStore core is composed of two main packages:
+1. [`store-ui`](https://storeui.netlify.app/): A performant ecommerce ready component library
+2. `store-sdk`: An SDK for ecommerce to solve your basic ecommerce needs, like cart, pixel management, session etc.
+
+These packages have bindings to popular frameworks, like Gatsby. These bindings are listed in this monorepo and help you create stores with Multiregion, Internationalization etc.
+We want to support other frameworks, like Next.JS and Create React App in the future
```
Comment (tlgimenes) [inline README.md:57]:
```suggestion
We also want to support other React frameworks in the future, such as Next.JS, or even React applications bootstrapped using Create React App"
```

**Review by lariciamota — APPROVED** [review]

File: README.md [hunk 8]
```diff
@@ -44,14 +44,21 @@ The priorities are all about the stores' user experience and retailer satisfacti
...
-## Why Jamstack and Gatsby?
+## Basic Architecture
 
-Jamstack focuses on performance and stability, which is part of our core philosophy.
+FastStore core is composed of two main packages:
+1. [`store-ui`](https://storeui.netlify.app/): A performant ecommerce ready component library
+2. `store-sdk`: An SDK for ecommerce to solve your basic ecommerce needs, like cart, pixel management, session etc.
+
+These packages have bindings to popular frameworks, like Gatsby. These bindings are listed in this monorepo and help you create stores with Multiregion, Internationalization etc.
+We also want to support other React frameworks in the future, such as Next.JS, or even React applications bootstrapped using Create React App"
```
Comment (victorhmp) [inline README.md:57]:
```suggestion
We also want to support other React frameworks in the future, such as Next.JS, or even React applications bootstrapped using Create React App
```

**Review by victorhmp — APPROVED** [review]
Awesome!

File: README.md [hunk 9]
```diff
@@ -7,7 +7,7 @@
   - [An open ecommerce framework](#an-open-ecommerce-framework)
   - [Why building FastStore?](#why-building-faststore)
   - [Project Philosophy](#project-philosophy)
```
Comment (tlgimenes) [inline README.md:9]: - [Project Philosophy](#project-philosophy)
- [Basic Architecture](#basic-architecture)

**Pull request merged**